src/components/SimpleSlider/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import Slider from "react-slick";
+import './index.scss'
+
+export default class SimpleSlider extends Component {
```

Comment:
Si lo hacemos como en los demás componentes? 

```suggestion
class SimpleSlider extends Component { ... } 

export default SimpleSlider
```

Reply:
Igualmente.. este podría ser un componente simple 
```suggestion
const SimpleSlider = (props) => { 
const settings = {...} 
return (...) 
}
```